style.css

```diff
@@ -0,0 +1,280 @@
+*{
+	margin: 0;
+	padding: 0;
+}
+.navbar-brand{
+    font-size: 1.6rem !important;
```

Comment:
Using !important to override CSS declarations is considered bad practice because it can make debugging issues extra difficult. Please look for a better way to make this declaration apply without using !important.